src/odemis/acq/fastem.py

```diff
@@ -285,7 +286,7 @@ class AcquisitionTask(object):
     """
 
     def __init__(self, scanner, multibeam, descanner, detector, stage, ccd, beamshift, lens, roa, path,
-                 pre_calibrations, future):
+                 pre_calibrations, future, settings_obs=None):
```

Comment:
Update the doc string.

Reply:
Is it needed that it is None here? The acquire method already makes sure it is either None or an object right?

Reply:
you are correct, it doesn't matter here